code/core/src/manager/components/sidebar/ContextMenu.tsx

```diff
@@ -26,12 +26,16 @@ const empty = {
   node: null,
 };
 
-const FloatingStatusButton = styled(StatusButton)({
-  background: 'var(--tree-node-background-hover)',
-  boxShadow: '0 0 5px 5px var(--tree-node-background-hover)',
-  position: 'absolute',
-  right: 0,
-  zIndex: 1,
+const FloatingStatusButton = styled(StatusButton)(({ theme }) => {
+  // --tree-node-background-hover, but opaque to avoid layering colors
+  const background = theme.base === 'dark' ? '#1b2432' : '#ddebfa';
```

Comment:
This feels fragile, but I don't have a better option in the near term.

Reply:
I tried CSS color functions but they're very clunky and didn't produce the correct color in dark mode: `color-mix(in srgb, ${theme.background.app}, rgb(from var(--tree-node-background-hover) r g b / 1) 10%)`. It relies on knowing the original alpha value (10%) which doesn't even make this fully flexible. I scrapped it in the end. We should revisit this for theming 2.0.